docs/build/tutorials/platform/subnets/setup-dfk-node.md

```diff
@@ -104,3 +104,9 @@ cd subnet-evm
 ./scripts/build.sh $GOPATH/src/github.com/ava-labs/avalanchego/build/plugins/mDV3QWRXfwgKUWb9sggkv4vQxAQR4y2CyKrt5pLZ5SzQ7EHBv
 ./build/avalanchego --whitelisted-subnets Vn3aX6hNRstj5VHHm63TCgPNaeGnRSqCYXQqemSqDd2TQH4qJ
 ```
+
+## Appendices
+
+### Websocket Support on DFK Subnet
```

Comment:
I think this should be placed next to the other API metadata for DFK? I don't think the end of this tutorial is the right place unless we include all this info: 
<img width="586" alt="image" src="https://user-images.githubusercontent.com/13023275/164249919-61741ed8-3c31-453a-ad88-74ca07a83826.png">

Reply:
The URLs are displayed on 
https://subnets.avax.network/defi-kingdoms/dfk-chain/explorer 
https://subnets.avax.network/defi-kingdoms/dfk-chain-testnet/explorer.

We don't need to add in this doc.